### PR TITLE
fix: Add WASI 20 `_initialize` call to `acvm_backend.wasm` binary

### PR DESCRIPTION
**Pull request by kevaundray**
# Description

## Problem\*

Resolves 

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: acvm_js/test/node/execute_circuit.test.ts [hunk 1]
```diff
@@ -205,3 +205,31 @@ it("successfully executes two circuits with same backend", async function () {
   expect(solvedWitness0).to.be.deep.eq(expectedWitnessMap);
   expect(solvedWitness1).to.be.deep.eq(expectedWitnessMap);
 });
+
+it.only("successfully executes 500 circuits with same backend", async function () {
```
Comment (kevaundray) [inline acvm_js/test/node/execute_circuit.test.ts:209]: Would be good to keep a test like this for regression purposes without the console.log statements

File: acvm_js/test/node/execute_circuit.test.ts [hunk 2]
```diff
@@ -205,3 +205,29 @@ it("successfully executes two circuits with same backend", async function () {
...
```
Comment (LHerskind) [inline acvm_js/test/node/execute_circuit.test.ts:209]: You probably don't want the `only` in here?

Comment (kevaundray) [reply acvm_js/test/node/execute_circuit.test.ts]: Good point, just fixed

**Review by LHerskind — APPROVED** [review]
LGTM

**Pull request merged**